pkg/crypto/encrypt.go

```diff
@@ -14,31 +16,34 @@ var (
 	// ErrCipherTooShort occurs when `Decrypt` does not
 	// have input of enough length to decrypt using AES256
 	ErrCipherTooShort = errors.New("crypto: cipher plainText is too short for AES encryption")
+	// ErrCorruptedMessage occurs when an attempt of unsealing a message
+	// does not pass the authentication check
+	ErrCorruptedMessage = errors.New("crypto: the message didn't pass the authentication check")
 )
 
 // PassphraseToKey converts a string to a key for encryption.
 //
 // This function must be used STRICTLY ONLY for generating
 // an encryption key out of a passphrase.
 // Please don't use this function for hashing user-provided values.
-// It uses SHA2 for simplicity but it's slower. User-provided data should use SHA3
-// because of its better performance.
+// It uses SHA2 for simplicity and it's faster but less secure than SHA3.
```

Comment:
this claim was incorrect, SHA3 is slower according to https://en.wikipedia.org/wiki/SHA-3#Comparison_of_SHA_functions